src/views/Compose.vue

```diff
@@ -111,33 +271,270 @@
         </b-container>
       </b-tab>
     </b-tabs>
+    <ocs-custom-modal :show="showEdPopup" :show-cancel="false" @close="closeEdPopup" @submit="closeEdPopup">
+      <h3>Welcome to the LCO observation request page!</h3>
+      <p>Using this form you can instruct the LCO telescope network to perform an astronomical observation on your behalf.</p>
+      <p>
+        Fields should be filled out from top to bottom. Some fields labels have blue question marks next to them. Hover over one of these question
+        marks to view more information about that field.
+      </p>
+      <p>
+        A field highlighted in red with means that there is a problem with the given value. An error message will be displayed below any underneath a
+        field such as this. An observation request cannot be submitted until the form is free of errors.
+      </p>
+      <p>
+        Some elements may be copied using the <i class="fa fa-copy text-success" /> copy button. For example: to create an RGB image you can copy the
+        instrument configuration twice so that there are three, and set the filters in each accordingly.
+      </p>
+      <p>Thanks for using Las Cumbres Observatory!</p>
+    </ocs-custom-modal>
   </b-container>
 </template>
 <script>
-import moment from 'moment';
 import _ from 'lodash';
 import $ from 'jquery';
+import moment from 'moment';
 import Vue from 'vue';
+import { OCSUtil } from 'ocs-component-lib';
 
-import Requestgroup from '@/components/Requestgroup.vue';
-import Drafts from '@/components/Drafts.vue';
-import Sidenav from '@/components/Sidenav.vue';
-import CustomAlert from '@/components/util/CustomAlert.vue';
-import { datetimeFormat } from '@/utils.js';
+import Archive from '@/components/Archive.vue';
+import InstrumentConfigForm from '@/components/InstrumentConfigForm.vue';
+import { siteToColor, siteCodeToName, tooltipConfig, julianToModifiedJulian, lampFlatDefaultExposureTime, arcDefaultExposureTime } from '@/utils.js';
 
 export default {
   name: 'Compose',
   components: {
-    Requestgroup,
-    Drafts,
-    Sidenav,
-    CustomAlert
+    Archive,
+    InstrumentConfigForm
+  },
+  filters: {
+    generateDurationString: function(value) {
+      return OCSUtil.generateDurationString(value.duration);
+    }
   },
   data: function() {
+    let datetimeFormat = 'YYYY-MM-DD HH:mm:ss';
+    let simpleInterface = this.$store.state.profile.profile.simple_interface;
     return {
       tab: 1,
+      alerts: [],
+      instruments: {},
+      datetimeFormat: datetimeFormat,
+      siteToColor: siteToColor,
+      siteCodeToName: siteCodeToName,
+      tooltipConfig: tooltipConfig,
       draftId: -1,
-      requestgroup: {
+      targetLookup: {
+        busy: false,
+        failed: false,
+        text: '',
+        request: undefined
+      },
+      instrumentCategoryToName: {
+        SPECTRA: 'Spectrum',
+        IMAGE: 'Image'
+      },
+      formConfig: {
```

Comment:
Food for thought:

Following the general pattern of separating configuration from source code, do you think it might be better to separate the formConfig (and other config) out into their own config-type files? It would definitely work to trim down the amount of "stuff" in this component.

This writeup highlights a couple of tools that look like they're pretty good for runtime and environment-aware configuration, for what it's worth. https://read.humanjavascript.com/ch12-settings-and-configs.html

Reply:
Thanks for clarifying - I think my initial comment was kind of vague. Sorry about that. And yes, the article I sent definitely was dealing more with runtime/environment-driven config. 

I didn't necessarily have anything specific in mind - I think something like a configuration factory to generate some config _could_ be a nice enhancement, but totally not necessary for this PR. I see what you're saying about the proliferation of config files - so I completely agree that it would be a dangerous road to go down.

My main thought was "hmm, there's a lot of config here - I wonder if it _needs_ to be in the component?" But the argument could be made that since you're utilizing the upstream component lib, then you're trading actually having to write the code for just having to configure the general component you're using (which is a win in my book!). And so more config/less code makes perfect sense.

This is a tough one - I am kind of going back and forth on it, tbh. I think the PR is great as it is. I'll have to do some research on how this is handled in Vue/JS projects in general if I want to be any more opinionated!

Reply:
Yeah, for this particular case I also don't see a clear winner in the current codebase between leaving it as is and making a factory function in another file... Since we're both going back and forth on that, my vote is to leave it as is until need arises if that sounds OK to you.

Reply:
Yep, sounds great!